docs/reference/cli.mdx

```diff
@@ -0,0 +1,130 @@
+The Nitric CLI allows you to build, develop, and deploy your serverless application.
+
+To verify your installation run `nitric version`. The output should look something like this:
+
+```
+Go Version: go1.17.8
+Go OS/Arch: darwin/amd64
+Git commit: fce97c9
+Build time: 2022-03-23T14:05:59+1100
+Nitric CLI: v1.1.0-develop.14-98-gfce97c91c5bce6eeb864194e592134322b52d8e5
+```
+
+To get a list of the available commands, run the following command:
+
+```
+nitric help
+```
+
+## Common Commands
+
+There are a few common commands that you will use frequently:
+
+- `nitric new` creates a new project
+- `nitric stack new` creates a new stack within a project
+- `nitric run` runs your stack locally
+- `nitric up` deploys your stack to the cloud
+- `nitric down` destroys your stack from the cloud
+
+## Environment Variables
+
+### Pulumi Access Token
+
+To view your deployments on the pulumi dashboard, you will need a pulumi access token. You can get this token by logging into pulumi on the browser and going to your profile settings. Under the 'Access Tokens' tab click 'Create token'.
+
+Add the environment variable to your `~/.zshrc` or `~/.bashrc` as:
+
+```
+PULUMI_ACCESS_TOKEN=access_token
+```
+
+## Start
+
+### Projects
+
+To start a new project with nitric, run the `nitric new` command.
+
+```
+nitric new
+```
+
+This will walk through creating the project, including the name and what template you want to start from.
+
+The command will also create a `nitric.yaml` file. This file contains the configuration for your project. The handlers key is list of globs for your functions. The handlers should point to all functions you want to run or deploy.
+
+### Stacks
+
+Creating a new stack will create a configuration for a particular cloud. You can have multiple stacks for one project.
+
+```
+nitric stack new
+```
+
+This will create a `nitric-stackName.yaml` file that contains the configuration for deploying to the cloud. Some providers will have different config, heres an example of an aws stack
+
+```yaml
+# nitric-my-aws-stack.yaml
+name: my-aws-stack
+provider: aws
+region: us-east-1
+```
+
```

Comment:
Might be worth adding a note telling the user that this file is not the aws_credentials configuration, they still need that configured.

Reply:
Good suggestion, that would have been confusing.